mutiny-core/Cargo.toml

```diff
@@ -0,0 +1,77 @@
+cargo-features = ["per-package-target"]
+
+[package]
+name = "mutiny-core"
+version = "0.2.4"
+edition = "2021"
+authors = ["Tony Giorgio <tony@mutinywallet.com>", "benthecarman <ben@mutinywallet.com>"]
+forced-target = "wasm32-unknown-unknown"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+ln-websocket-proxy = "0.3.0"
+lnurl-rs = { version = "0.2", default-features = false, features = ["async", "async-https"] }
+
+cfg-if = "1.0.0"
+wasm-bindgen = "0.2.83"
+serde-wasm-bindgen = "0.5.0"
+bip39 = { version = "1.0.1" }
+bip32 = "0.4.0"
+bitcoin-bech32 = "0.12"
+js-sys = "0.3.60"
+secp256k1 = "0.24.0"
+bitcoin_hashes = { version = "0.11", default-features = false }
+bitcoin = { version = "0.29.2", default-features = false, features = ["serde", "secp-recovery"] }
+# TODO waiting for esplora version 0.4.0
+bdk = { git = "https://github.com/mutinywallet/bdk", branch = "esplora-tmp", default-features = false, features = ["keys-bip39", "esplora", "use-esplora-reqwest", "async-interface"] }
```

Comment:
yesterday bdk released 0.28 so we shouldn't need the fork, should be fine to do that here

Reply:
I'll leave for a seperate PR, I don't want to break anything